### PR TITLE
chore: upgrade to vue 2.6

### PR DESCRIPTION
**Pull request by Shigma**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [x] Other, please describe: Update dependencies

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by f3ltron** [comment]
i will have a look about changes  but it looks really great. I will resolve conflicts if you dont @Shigma

File: packages/@vuepress/theme-blog/layouts/Layout2.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<template>
```
Comment (kefranabg) [inline packages/@vuepress/theme-blog/layouts/Layout2.vue]: What is this `Layout2` component about?

File: packages/@vuepress/test-utils/package.json [hunk 2]
```diff
@@ -33,6 +33,10 @@
     "vue-jest": "^4.0.0-beta.1",
     "vue-template-compiler": "^2.5.16"
   },
+  "peerDependencies": {
```
Comment (kefranabg) [inline packages/@vuepress/test-utils/package.json]: Why now would we need these two dependencies as peerDependencies?

File: packages/@vuepress/markdown/index.js [hunk 3]
```diff
@@ -20,7 +20,9 @@ const tocPlugin = require('markdown-it-table-of-contents')
 const {
   slugify: _slugify,
   parseHeaders,
-  logger, chalk, normalizeConfig,
```
Comment (kefranabg) [inline packages/@vuepress/markdown/index.js]: Could you revert the changes about formatted code?

File: packages/@vuepress/test-utils/package.json (hunk 2 above)
Comment (f3ltron) [reply packages/@vuepress/test-utils/package.json]: in test utils we no mention of vuejs. It's just a protection to say i need at least vue 2x to work. It installing nothing

File: packages/@vuepress/theme-blog/layouts/Layout2.vue (hunk 1 above)
Comment (f3ltron) [reply packages/@vuepress/theme-blog/layouts/Layout2.vue]: yeah it seem useless... i didnt pay attention on it

File: packages/@vuepress/markdown/index.js (hunk 3 above)
Comment (f3ltron) [reply packages/@vuepress/markdown/index.js]: it is a format from the code it should happen i think

Comment (kefranabg) [reply packages/@vuepress/markdown/index.js]: I don't see any config for code format in the vuepress repos. In this case, I think we should not do changes about code formatting in PRs to avoid unnecessary changes.

Comment (f3ltron) [reply packages/@vuepress/markdown/index.js]: i will revert that change yeah

**Comment by f3ltron** [comment]
As you create a new one i close this one https://github.com/vuejs/vuepress/pull/1370

**Comment by f3ltron** [comment]
As you create a new one i close this one https://github.com/vuejs/vuepress/pull/1370
